config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh

```diff
@@ -446,13 +448,51 @@ periodics:
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     testgrid-num-columns-recent: '30'
 
+- interval: 24h
+  name: capz-conformance-${release/./-}
+  decorate: true
+  decoration_config:
+    timeout: 3h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-azure-cred: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-azure
+    base_ref: master
+    path_alias: sigs.k8s.io/cluster-api-provider-azure
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210412-176e4b6-master
+      command:
+      - runner.sh
+      - ./scripts/ci-conformance.sh
+      env:
+      - name: E2E_ARGS
+        value: "-kubetest.use-ci-artifacts"
+      - name: KUBERNETES_VERSION
+        value: "${kubernetes_version}"
+      - name: CONFORMANCE_WORKER_MACHINE_COUNT
```

Comment:
I might be wrong but I thought conformance required 5+ worker nodes?

Reply:
I can't seem to find anything that mentions this requirement. The reason why I set it to 2 is that
- I want to minimize the core consumption
- existing conformance test on Azure are running on a 2-node cluster

Reply:
SGTM, I might be misremembering